packages/examples/jest.config.ts

```diff
@@ -0,0 +1,21 @@
+import config from "../../scripts/config/jest.config";
+
+export default {
+  ...config,
+  transform: {
+    "\\.(gql|graphql)$": "@graphql-tools/jest-transform",
+    // Needed for packages/examples to load relay compiler generated artifacts.
+    // It would be better if it could be config local to examples package but haven't found a way to configure it with just
```

Comment:
It looks like it is local to the examples package? Am I missing something or does the comment need to be updated?

Reply:
Right, I mindlessly copy pasted, will remove on next PR